app/Models/Tenants/Product.php

```diff
@@ -46,9 +46,9 @@ public function CartItems(): HasMany
 
     public function scopeStockLatestCalculateIn()
     {
-        $usingFifoPrice = Setting::get('selling_method', env('SELLING_METHOD', 'fifo')) == 'fifo';
-        $usingNormalPrice = Setting::get('selling_method', env('SELLING_METHOD', 'fifo')) == 'normal';
-        $usingLifoPrice = Setting::get('selling_method', env('SELLING_METHOD', 'fifo')) == 'lifo';
+        $usingFifoPrice = Setting::get('selling_method', config('method.selling_method', 'fifo')) == 'fifo';
```

Comment:
Sweet!

Reply:
I already added test for config that i created